keyboards/runner3680/5x6/keymaps/default/keymap.c

```diff
@@ -0,0 +1,111 @@
+#include QMK_KEYBOARD_H
+#include "action_layer.h"
+#include "eeconfig.h"
```

Comment:
Could you remove all of these additional includes?  

```suggestion
```

They get included automatically, at various points in the built process, so they are redundant and not necessary here.